dependency-map.json

```diff
@@ -497,7 +497,7 @@
   },
   "polymer": {
     "npm": "@polymer/polymer",
-    "semver": "^3.0.0-pre.10"
+    "semver": "3.0.0-pre.12"
```

Comment:
Was removing the leading ^ intentional here?

Reply:
yes, OP has been updated with the rational for the change